test/__tests__/data/customerio_input.json

```diff
@@ -2614,7 +2682,6 @@
       "request_ip": "[::1]",
       "rudderId": "0aef312c-0dc0-4a49-b613-4f33fb4e9b46",
       "sentAt": "2022-01-10T10:00:26.982Z",
```

Comment:
shouldn't the right timestamp come here ? Instead how it's being omitted from the payload ?

Reply:
We should revisit this, as the term `historical timestamp` seems very particular to customerIO here @anantjain45823

Reply:
> shouldn't the right timestamp come here ? Instead how it's being omitted from the payload ?

We have test cases for all type of timestamps following ISO format or timestamp as a number

Reply:
> We should revisit this, as the term `historical timestamp` seems very particular to customerIO here @anantjain45823

We are not changing the `historical timestamp` but the `created_at` field which excepts much old date as well